src/volume/volume_sphere.cpp

```diff
@@ -0,0 +1,89 @@
+/*
+	Lightmetrica - Copyright (c) 2019 Hisanari Otsu
+	Distributed under MIT license. See LICENSE file for details.
+*/
+
+#include <pch.h>
+#include <lm/core.h>
+#include <lm/volume.h>
+
+LM_NAMESPACE_BEGIN(LM_NAMESPACE)
+#define SQ3 1.73205080757
```

Comment:
Please avoid to use macros for a constant. Also please give a context with comment or clear naming. It seems this constant is used only from a function, then you can define the constant inside the function.